e2e/page-objects/viewIssue.po.ts

```diff
@@ -0,0 +1,38 @@
+import { browser, by, element } from 'protractor';
+
+export class ViewIssuePage {
+
+  async enterNewIssueTitle(title: string) {
+    return element(by.id('title')).sendKeys(title);
+  }
+
+  async enterNewBugReportDescription(desc: string) {
+    const textArea = element(by.className('text-input-area'));
+    await textArea.clear();
+    return textArea.sendKeys(desc);
+  }
+
+  async selectSeverityDropdown() {
+    return element(by.className('severity-dropdown')).click()
+      .then(() => browser.sleep(100)); // Allow time for DOM to update. (Dropdown Selection may fail if DOM is stale)
```

Comment:
Instead of a fixed sleep time, can we ask Protractor to wait till the dropdown option is clickable?
[Link to relevant Protractor API](https://www.protractortest.org/#/api?view=ProtractorExpectedConditions.prototype.elementToBeClickable)

Reply:
Yes I tried, but have not been able to consistently get it to work.

```typescript
await browser.wait(ExpectedConditions.presenceOf(element(by.className('mat-option'))));
```

I can make potractor wait for the `mat-option` but I would have to add a comment to not string multiple drop-down + option click actions. Should I put that in the dev notes or in the `viewIssue.po.ts`? I think leaving it in `viewIssue` would be better since its pretty specific to `viewIssue` page.

Reply:
I have added a comment in the dropdown selection code, commit message and PR description for reference

Reply:
I'm not very clear on what you mean by "the data fetched by the driver was stale".
Can you illustrate the issue that would occur if we string 2 dropdown + option clicks together?
The tests worked fine on my machine, when I tried this.